PHPCI/Command/DaemonCommand.php

```diff
@@ -40,85 +49,133 @@ protected function configure()
             ->setName('phpci:daemon')
             ->setDescription('Initiates the daemon to run commands.')
             ->addArgument(
-                'state',
-                InputArgument::REQUIRED,
-                'start|stop|status'
+                'action', InputArgument::REQUIRED, 'start|stop|status'
+            )
+            ->addOption(
+                'pid-file', 'p', InputOption::VALUE_REQUIRED,
+                'Path of the PID file',
+                PHPCI_DIR . 'daemon' . DIRECTORY_SEPARATOR . 'daemon.pid'
             );
     }
 
     /**
-    * Loops through running.
-    */
+     * Loops through running.
+     */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $state = $input->getArgument('state');
+        if(!function_exists('pcntl_fork') || !function_exists('posix_setsid')) {
```

Comment:
I'm not sure I am comfortable with adding a new dependency (pcntl) for something that worked fine before (without the dependency).